ms/MSOper/MSMetaData.cc

```diff
@@ -4942,11 +4968,59 @@ MSMetaData::ColumnStats MSMetaData::getIntervalStatistics() const {
     return stats;
 }
 
+std::shared_ptr<vector<int>> MSMetaData::_almaReceiverBands(uint nspw) const {
+    // for ALMA specific receiver band (RB) info
```

Comment:
Maybe add a few more lines of comments on how the the receiver bands are expected to be defined in the ASDM_RECEIVER table and where this table comes from (i.e. that we are expecting importasdm to have copied it over with the "asis" feature verbatim from the ASDM). After all, ASDM_RECEIVER is not a standard MS subtable.